core/processor/src/main/java/io/quarkus/annotation/processor/Constants.java

```diff
@@ -67,7 +67,8 @@
             SYSTEM_PROPERTIES.getProperty("maven.multiModuleProjectDirectory", "."));
     public static final Path GENERATED_DOCS_PATH = Paths.get(DOCS_OUT_DIR + DOCS_SRC_MAIN_ASCIIDOC_GENERATED).toAbsolutePath();
     public static final File GENERATED_DOCS_DIR = GENERATED_DOCS_PATH.toFile();
-    public static final Boolean SKIP_DOCS_GENERATION = Boolean.valueOf(SYSTEM_PROPERTIES.getProperty("skipDocs", "false"));
+    public static final Boolean SKIP_DOCS_GENERATION = Boolean.parseBoolean(SYSTEM_PROPERTIES.getProperty("skipDocs", "false"))
+            || Boolean.parseBoolean(SYSTEM_PROPERTIES.getProperty("quickly", "false"));
```

Comment:
should be `"true"` ?

Reply:
ah, sorry that's the default.

Reply:
Currently skipping the documentation generation is opt-in, so we don't want to skip by default

Reply:
yes, sure I got confused by the API :)

Reply:
:)